index.js

```diff
@@ -214,5 +220,56 @@ module.exports = {
       'space-unary-ops': 'error',
       'unicode-bom': 'error',
 
+      'arrow-spacing': [ 'error', { 'before': true, 'after': true } ],
    },
+
+   'overrides': [
+      {
+         'files': [ '*.ts' ],
+         'parser': 'typescript-eslint-parser',
+         'parserOptions': {
+            'sourceType': 'module',
+            // Disable warning banner for possibly incompatible versions of TypeScript
+            'loggerFn': false,
```

Comment:
FYI to reviewers: This property is only used by `typescript-eslint-parser`. It will not affect any other parsing.